build-tools/scripts/Version.props.in

```diff
@@ -1,8 +1,15 @@
 <Project>
   <PropertyGroup>
     <Version Condition=" '$(Version)' == '' ">@VERSION@</Version>
-    <InformationalVersion>@VERSION@ git-rev-head:@COMMIT@ git-branch:@BRANCH@</InformationalVersion>
     <Company>Microsoft Corporation</Company>
     <Copyright>Microsoft Corporation</Copyright>
   </PropertyGroup>
-</Project>
\ No newline at end of file
+
+  <Target Name="_SetInformationalVersion"
+      BeforeTargets="GetAssemblyVersion;GetPackageVersion">
```

Comment:
This looks right: https://github.com/dotnet/sdk/blob/2a515cdbd8f6be1b019ae2c8d7f21952592f0697/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets#L205-L214

We might want to manually check the `Java.Interop.dll` in the next java.interop bump? Will we remember?